samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/services/assessment/AssessmentEntityProducer.java

```diff
@@ -453,25 +472,26 @@ public void updateEntityReferences(String toContext, Map<String, String> transve
 							continue;
 						}
 
+						// TODO: why are these copyAttachments = false?
```

Comment:
While I can't answer the _why_ definitely, what `copyAttachments` does, did not happen for instructions and descriptions, so that boolean exists to keep the previous behavior. My focus was on improving performance while maintaining functionality, I didn't want to add more process time, especially without fully understanding this _why_.

Reply:
I feel the same - it looks like `copyAttachments` makes _another_ copy of embedded things like images attachments and only is used for the questions and answers and not the assessment-level text - so I will follow that pattern as well.